lntest/node.go

```diff
@@ -584,6 +600,30 @@ func (hn *HarnessNode) start(lndBinary string, lndError chan<- error,
 		}
 	}
 
+	getFinalizedLogFilePrefix := func() string {
+		pubKeyHex := hex.EncodeToString(
+			hn.PubKey[:logPubKeyBytes],
+		)
+
+		return fmt.Sprintf("%s/%d-%s-%s-%s",
+			GetLogDir(), hn.NodeID,
+			hn.Cfg.LogFilenamePrefix,
+			hn.Cfg.Name, pubKeyHex)
+	}
+
+	finalizeEtcdLog := func() {
```

Comment:
A lot of this logic is shared with `finalizeLogfile`, could we just add one `renameLog` function which takes the `fineName` and an identifier string (`etcd.log` or `.log`) and share rather than making a copy of it? Makes it easier to update the nodeID/prefix/name/pubkey format in once place if we want to.

Reply:
I tried doing this by extracting the `getFinalizedLogFilePrefix` function above and creating a separate functon for `renameFile`. I was thinking whether it's worth adding more generalization but since these are not used elsewhere I meant to just minimally reorder and make things more readable. The main pain point imo is the path construction and that at least is now more readable. wdyt?

Reply:
Was thinking that we could combine the two to a single closure, but non-blocking since it's not saving us much. Agree that `getFinalizedLogFilePrefix` addresses most of the inconvenience.